pages/about.tsx

```diff
@@ -2,8 +2,8 @@ import React from 'react'
 import Container from '@material-ui/core/Container'
 import Box from '@material-ui/core/Box'
 import MuiLink from '@material-ui/core/Link'
+import Link from 'next/link'
```

Comment:
Tempted to import next/link as NextLink instead of Link, as per importing mui/core/link as MuiLink